src/main/java/io/pravega/perf/PravegaPerfTest.java

```diff
@@ -414,4 +430,88 @@ private void shutdown(long endTime) {
             return readers;
         }
     }
+
+    static private class KafkaTest extends Test {
+        final private Properties producerConfig;
+        final private Properties consumerConfig;
+
+        KafkaTest(long startTime, CommandLine commandline) throws
+                IllegalArgumentException, URISyntaxException, InterruptedException, Exception {
+            super(startTime, commandline);
+            producerConfig = createProducerConfig();
+            consumerConfig = createConsumerConfig();
+        }
+
+        private Properties createProducerConfig() {
+            if (producerCount < 1) {
+                return null;
+            }
+            final Properties props = new Properties();
+            props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, controllerUri);
+            props.put(ProducerConfig.ACKS_CONFIG, "all");
+            props.put(ProducerConfig.RETRIES_CONFIG, 0);
```

Comment:
ok, I am not setting the ProducerConfig.RETRIES_CONFIG now , it will have the default value of Integer MAX.
but , enable.idempotence is required for transactions so I will raise the separate PR for kafka transactions support and comparison with Pravega.